packages/auth/lib/Users.js

```diff
@@ -173,11 +173,13 @@ const signIn = async (_email, context, pgdb, req, consents, _tokenType) => {
     })
     if (shouldAutoLogin({ email })) {
       setTimeout(async () => {
-        console.log('AUTO_LOGIN!')
+        console.warn(`🔓💥 auto login for ${email}!!! AUTO_LOGIN_REGEX`)
         await authorizeSession({
           pgdb,
           tokens: [token],
-          email
+          email,
+          req,
```

Comment:
Was this broken before adding `req` and `me`?

Reply:
yes, and therefore the tests as well